<?xml version="1.0" encoding="ISO-8859-15"?>
<launch>
  <node name="voxblox_node" pkg="voxblox_ros" type="esdf_server" output="screen" args="-alsologtostderr" clear_params="true">
    <remap from="pointcloud" to="/firefly/vi_sensor/camera_depth/depth/points"/>
    <param name="use_freespace_pointcloud" value="false" />
    <param name="tsdf_voxel_size" value="0.2" />
    <param name="tsdf_voxels_per_side" value="16" />
    <param name="voxel_carving_enabled" value="true" />
    <param name="color_mode" value="colors" />
    <param name="use_tf_transforms" value="false" />
    <param name="verbose" value="true" />
    <remap from="transform" to="firefly/odometry_sensor1/transform" />
    <param name="update_mesh_every_n_sec" value="1.0" />
    <param name="slice_level" value="0.2" />
    <param name="method" value="fast" />
    <param name="use_const_weight" value="false" />
    <param name="publish_slices" value="true" />
    <param name="publish_pointclouds" value="true" />

    <rosparam file="$(find voxblox_ros)/cfg/euroc_dataset.yaml"/>

    <param name="mesh_filename" value="$(find voxblox_ros)/mesh_results/$(anon euroc).ply" />
  </node>
</launch>
